.github/workflows/actions.yml

```diff
@@ -2,6 +2,9 @@ name: MapSwipe Workers
 
 on: [push]
 
+env:
+  COMPOSE_FILE: docker-compose.yaml:docker-compose-ci.yaml
+
```

Comment:
@thenav56 Can you verify this usage?

Reply:
Yes, You can check this for more detail https://github.com/mapswipe/python-mapswipe-workers/pull/962/commits/228a85ba60a409dded84b817cf6cccd1eb3fa1e9